backend-server/src/api/routes/availability.ts

```diff
@@ -7,6 +7,9 @@ import {
 } from "../controllers/availabilityController";
 
 export const availabilityRouter = express.Router();
+// enable cors
```

Comment:
I'm going to move this to `app.ts` so we don't have to add this middleware to each router

Reply:
I think @SarahBornais and I tried that when we were debugging the CORS error but it wouldn't work for some reason when it was in app.ts...unless we were missing something?

Reply:
Hmm ok, I moved it in my last commit, maybe give it another try to see if it's working for you?